scripts/select_appointments.sql

```diff
@@ -1 +1,18 @@
 -- retrieve appointment information
+SELECT p.given_name, d.surname, a.appointment_time
+FROM patients p
+INNER JOIN doctors d
+  ON d.doctor_id = d.id
+INNER JOIN appointments a
+  ON a.appointment_id = a.id
```

Comment:
Since `appointments` is the join, we need to link it both to `patients` and `doctors` by comparing the `doctor_id` within `appointments` to ids of rows in `doctors`, ditto 'patients`.